packages/react-server/src/ReactFlightServer.js

```diff
@@ -235,7 +235,12 @@ function performWork(request: OpaqueRequest): void {
   }
 }
 
+let reentrant = false;
 function flushCompletedChunks(request: OpaqueRequest): void {
+  if (reentrant) {
+    return;
+  }
+  reentrant = true;
```

Comment:
I don't understand what this is for.

Reply:
I had to hack it because I didn’t realize whatwg streams are reentrant. Will need a better refactor to support both types of a streams.